ajax/libs/sketchpad/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "sketchpad",
+  "version": "0.1.0",
+  "description": "A simple sketchpad.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/yiom/sketchpad.git"
+  },
+  "homepage": "https://yiom.github.io/sketchpad/",
+  "keywords": [
+    "sketchpad",
+    "javascript"
+  ],
+  "author": "Yiom <https://yiom.com.br>",
+  "license": "MIT",
+  "filename": "sketchpad.min.js",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/yiom/sketchpad.git",
+    "fileMap": [
+      {
+        "basePath": "scripts",
+        "files": [
+          "*.js"
```

Comment:
It's not a good way to use `*` to fetch all js files in a non-dist folder, let's turn to use `+(initialize|sketchpad)*.+(js|map)` and `sketchpad*.+(css|map)` for styles. Thanks.